app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
This is missing a count of total results found as requested by the user story. It's great that you are only showing the top 10, but the user story also asks for a count of total results found. For the example search phrase "I am", there should've been 151.